docs/README.md

```diff
@@ -0,0 +1,351 @@
+# Test and config files
+
+A typical `*-bench` project runs checks specified in `controls` files that are a YAML 
+representation of the CIS Benchmark checks. There is a 
+`controls` file per version and node type.
+
+`controls` for the various versions can be found in directories
+with same name as the CIS versions under `cfg/`, for example `cfg/cis-1.4.0`.
+`controls` are also organized by distribution under the `cfg` directory, for
```

Comment:
Also not sure that "organized by distribution" is generic, I would drop that sentence